views/projects.erb

```diff
@@ -0,0 +1,41 @@
+<html>
+<body>
+
+  <head>
+    <title>Projects Page</title>
+  </head>
+
+  <header>
+  <h1><img src="http://www.cliparthut.com/clip-arts/1778/cat-head-clip-art-1778708.jpg" height=25 width = 25>
+    Ricky's Projects
+  <img src="http://www.cliparthut.com/clip-arts/1778/cat-head-clip-art-1778708.jpg" height=25 width = 25></h1>
+  </header>
+
+  <section>
+
+    <h3>I love projects. Here are some examples:</h3>
+    <ol>
+      <li>Cooking</li>
+      <img src = "http://belarr.com/bakercat/bakercat.gif" height = 200 width = 200>
+      <li>Reading</li>
+      <img src = "/reading.jpg" height = 200 width = 200>
+      <li>Blogging</li>
+      <img src = "http://rs1187.pbsrc.com/albums/z393/KristineChavezRodriguez/blogger.gif~c200" height = 200 width = 200>
+    </ol>
+
+  </section>
+
+
+  <footer>
+    <table border = "1">
+      <tr>
+        <td><a href = "/index">Home Page</a></td>
+        <td><a href = "/about">About Me</a></td>
+        <td><a href = "/projects">My Projects</a></td>
+        <td><a href = "/blog">My Blog</a></td>
+    </table>
+  </footer>
+
```

Comment:
Nice job overall - I might only use one line of whitespace to logically separate items rather than two. Eventually it will just be easier to read that way